codegen/smithy-python-codegen/src/main/java/software/amazon/smithy/python/codegen/integration/HttpBindingProtocolGenerator.java

```diff
@@ -986,6 +986,11 @@ private void deserializeIndividualHeaders(
                     """, locationName, memberName, targetHandler);
                 }
             }
+            writer.write("""
+                    case _:
+                        pass
+
+                    """);
```

Comment:
It may be useful to note we're treating anything unknown as a no-op, this isn't the most obvious at first glance.